src/main/java/org/mule/extension/sftp/internal/source/SftpDirectoryListener.java

```diff
@@ -234,6 +234,7 @@ private SftpFileSystem openConnection() throws Exception {
     try {
       fileSystem.changeToBaseDir();
     } catch (Exception e) {
+      LOGGER.debug("Exception while trying to open connection. Cause: " + e.getCause() + ". Message: " + e.getMessage());
```

Comment:
How do we know that the cause will be relevant? getCause() returns a Throwable. If we want to log something from the cause it would probably be its message.

Reply:
We don't know if the cause will be relevant, maybe it is maybe it isn't. But I don't see how logging extra information can be harmful.

Reply:
Also, use {} please